cadquery/occ_impl/exporters/assembly.py

```diff
@@ -23,34 +25,62 @@
 from OCP.Message import Message_ProgressRange
 from OCP.Interface import Interface_Static
 
-from ..assembly import AssemblyProtocol, toCAF, toVTK
+from ..assembly import AssemblyProtocol, toCAF, toVTK, toFusedCAF
 from ..geom import Location
 
 
-def exportAssembly(assy: AssemblyProtocol, path: str, **kwargs) -> bool:
+class ExportModes:
+    DEFAULT = "default"
+    FUSED = "fused"
+
+
+STEPExportModeLiterals = Literal["default", "fused"]
+
+
+def exportAssembly(
+    assy: AssemblyProtocol,
+    path: str,
+    exportMode: STEPExportModeLiterals = "default",
+    **kwargs
+) -> bool:
     """
     Export an assembly to a STEP file.
 
     kwargs is used to provide optional keyword arguments to configure the exporter.
 
     :param assy: assembly
     :param path: Path and filename for writing
+    :param exportMode: STEP export mode. The options are default (the current _toCAF method),
+        and fused (a single fused compound). It is possible that fused mode may exhibit low performance.
     :param write_pcurves: Enable or disable writing parametric curves to the STEP file. Default True.
-
         If False, writes STEP file without pcurves. This decreases the size of the resulting STEP file.
     :type write_pcurves: bool
     :param precision_mode: Controls the uncertainty value for STEP entities. Specify -1, 0, or 1. Default 0.
         See OCCT documentation.
     :type precision_mode: int
+    :param fuzzy_tol: OCCT fuse operation tolerance setting used only for fused assembly export.
+    :type fuzzy_tol: float
+    :param glue: Glue setting used only for fused assembly export. Can be used to improve performance, but only for
+        non-overlapping shapes.
```

Comment:
I'd remove the statement on performance as I find improvement for both overlap and non-overlapping cases.

Reply:
Done

Reply:
I don't think this is correct. AFAICT `glue=True` only works correctly for non-overlapping solids. See https://dev.opencascade.org/content/gluing-options-boolean-component

Reply:
In my (limited) testing I found speed up in both overlapping and non-overlapping cases.  Though I now see my experiment is a simple case like figure 4 of the article.  

```py
for i in range(1, N+1, 1):
    for j in range(1, N+1, 1):
        assy.add(
            box,
            name=f"box{i}_{j}",
            color=cq.Color("red"),
            #loc=cq.Location((1.0 * i, 1.0 * j, 0)),
            loc=cq.Location((0.9 * i, 0.9 * j, 0)),
        )
```

Without glue (N=50), fuse op time was ~300s total assy save time ~320s.  With glue=True, fuse op ~72s, total ~94s.  So based on this testing I made the comment about the docstring as it did not appear to be correct.

Now testing a case more like figure 5 I see the fuse op does not work with `glue=True`.


To provide guidance in the docstring (or elsewhere in docs) on when to specify glue, perhaps something like this is more accurate: 

```
    :param glue: Enable gluing mode for improved performance during fused assembly export.
        This option should only be used for non-intersecting shapes or those that are only touching or partially overlapping.
        Note that when glue is enabled, the resulting fused shape may be invalid if shapes are intersecting in an incompatible way.
        Defaults to False.
```

Reply:
LGTM

Reply:
I applied your suggestion @lorenzncode